pages/1.14/deploying-services/quota-management/index.md

```diff
@@ -75,43 +74,47 @@ Note that deleting quota doesn't affect any running services inside the group. S
 
 
 ### Deploying Services
-Once quota is set on a group by an administrator, regular users can simply deploy their services in a group as usual. If `enforceRole` property is set on the group, the service will be automatically configured to use the group role and hence limited by the quota. If the property is not set, but a user still wishes that their service be limited by quota, they could configure their service with the group role manually.
-
-
+Once quota is set on a group by an administrator, regular users can simply deploy their services in a group as usual. If the `enforceRole` property is set on the group, the service will be automatically configured to use the group role and hence limited by the quota. If the property is not set, but users want their service be limited by a quota, they can configure their service with the group role manually.
 
 ### Migrating services
 
 For backwards compatibility with existing deployments, services launched in a group do not use the group role by default but rather use their legacy role as before (the actual legacy role depends on whether the service was launched by native marathon, non-native marathon, catalog service etc).
```

Comment:
Can we rephrase the first sentence to "For backwards compatibility, any existing and new groups will have `enforceRole` property set to false. Consequently, existing or new services launched in such groups continue to use their legacy role (actual role depends on the service) instead of group role."